### PR TITLE
Add yapf linting base and lint dataviz app

### PR DESCRIPTION
**Pull request by mbeacom**
##### Description
The goal of this PR is to introduce `yapf` for automatic python styling.  Additionally, the `dataviz` app has been automatically formatted with `yapf` to get us started.  We'll want to slowly migrate each django app over to yapf formatting to allow us to add the check to precommit/CI.

##### Checklist
- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)
backend

##### Testing
Tested locally.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -68,3 +68,18 @@ default_section=THIRDPARTY
...
+[yapf]
+based_on_style = pep8
+column_limit = 120
+indent_width = 4
+spaces_before_comment = 2
+ALLOW_SPLIT_BEFORE_DICT_VALUE = false
+DEDENT_CLOSING_BRACKETS = true
+EACH_DICT_ENTRY_ON_SEPARATE_LINE = true
+COALESCE_BRACKETS = true
+USE_TABS = false
```
Comment (thelostone-mc) [inline setup.cfg]: @mbeacom no tabs ?

**Comment by thelostone-mc** [comment]
@mbeacom thanks for your contribution :) :trollface:

**Comment by mbeacom** [comment]
🤗

File: setup.cfg (hunk 1 above)
Comment (mbeacom) [reply setup.cfg]: No tabs for you!

**Pull request merged**